GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Controllers/Account/ChangeEmailTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Models;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.Extensions;
+using GenderPayGap.WebUI.Controllers.Account;
+using GenderPayGap.WebUI.Models.Account;
+using GenderPayGap.WebUI.Services;
+using GenderPayGap.WebUI.Tests.Builders;
+using Microsoft.Extensions.Primitives;
+using NUnit.Framework;
+
+namespace GenderPayGap.WebUI.Tests.Controllers.Account
+{
+    [TestFixture]
+    [SetCulture("en-GB")]
+    public class ChangeEmailTests
+    {
+
+        [Test]
+        [Description("POST: Providing a valid new email address results in a verification email being sent")]
+        public void POST_Providing_A_Valid_New_Email_Address_Results_In_A_Verification_Email_Being_Sent()
+        {
+            // Arrange
+            User user = new UserBuilder().WithEmailAddress("old@test.com").Build();

```

Comment:
This is really minor, but I tend to use `@example.com` for fake email addresses:
* just in case we accidentally send the email address a real email
* because http://example.com is an official domain for use in documentation, so it there won't be a real person who might end up receiving our emails

Reply:
Ah that's interesting, I didn't know that! Will swap them over 😄